library/alloc/src/vec/mod.rs

```diff
@@ -3288,10 +3291,12 @@ impl<T, A: Allocator> From<Box<[T], A>> for Vec<T, A> {
 #[cfg(not(test))]
 #[stable(feature = "box_from_vec", since = "1.20.0")]
 impl<T, A: Allocator> From<Vec<T, A>> for Box<[T], A> {
-    /// Convert a vector into a boxed slice.
+    /// Converts a vector into [`Box<[T]>`][owned slice].
```

Comment:
What's the intent behind changing this line?

The [function signature already links](https://doc.rust-lang.org/nightly/std/boxed/struct.Box.html#method.from-23) to `Box` so spelling in plain english seems fine to me and happens to teach the reader an alternate way to call this.

Reply:
Ah, I was just making both doc comments line up with each other since I was already updating part of the From<Vec<...>> docs. I'll revert that specific change.